config/index.js

```diff
@@ -297,6 +297,23 @@ var conf = convict({
     format: Array,
     default: [],
     env: 'OPENID_PROVIDERS'
+  },
+  statsd: {
+    enabled: {
+      doc: 'enable UDP based statsd reporting',
+      default: true,
+      env: 'ENABLE_STATSD'
+    },
+    host: 'localhost',
+    port: {
+      format: 'port',
+      default: 8125
+    },
```

Comment:
Please give these env vars as well, I think pretty much *everything* in this file is settable by env var